src/mono/System.Private.CoreLib/src/System/Reflection/Emit/DerivedTypes.Mono.cs

```diff
@@ -453,22 +453,22 @@ protected override bool IsByRefImpl()
 
         public override Type MakeArrayType()
         {
-            throw new ArgumentException("Cannot create an array type of a byref type");
+            throw new ArgumentException(SR.Argument_CannotCreateArrayTypeOfByref);
         }
 
         public override Type MakeArrayType(int rank)
         {
-            throw new ArgumentException("Cannot create an array type of a byref type");
+            throw new ArgumentException(SR.Argument_CannotCreateArrayTypeOfByref);
         }
 
         public override Type MakeByRefType()
         {
-            throw new ArgumentException("Cannot create a byref type of an already byref type");
+            throw new ArgumentException(SR.Argument_CannotCreateArrayTypeOfByref);
```

Comment:
This should have a slightly different message, no?

Reply:
got it, ill add the change in name to:
Argument_CannotCreateByrefTypeOfExistingByrefType

in the next commit?

Reply:
Perhaps, but my point was the messages above both matched, but this one was different but is now the same. Just curious if this was intentional.

Reply:
Oh, my bad i added the message in wrong,
ill fix that and add in a better name for the SR

Reply:
Does the new commit fix this problem?